packages/blitz/src/types.ts

```diff
@@ -2,10 +2,34 @@ import {UrlObject} from "url"
 // Context for plugins to declaration merge stuff into
 export interface Ctx {}
 
+export interface MiddlewareCtx {}
+
```

Comment:
Because this is same as Ctx, so we need to use Ctx instead
```suggestion
```

Reply:
I tried changing the type to Ctx, but it causes type issues in the resolvers
![image](https://user-images.githubusercontent.com/83594610/203279829-6da9799b-5da9-4145-8909-c955f4dbf54e.png)
So I have reverted the changes in the latest commit

Reply:
ok I will investigate